samples/rag-storage-document-loaders/README.md

```diff
@@ -0,0 +1,39 @@
+# RAG Agent with AzureBlobStorageLoader Demo
+This demo creates a RAG agent that responds to queries based on documents loaded from Azure Blob Storage.
+
+## Quick Start
+1. **Install dependencies:**
+   ```bash
+   python -m venv .venv
+   ./.venv/Scripts/activate  # Windows only - Use `source .venv/bin/activate` on macOS/Linux
+   python -m pip install -r requirements.txt
+   ```
+
+2. **Configure environment variables:**
+   ```bash
+   dotenv set AZURE_STORAGE_ACCOUNT_URL "https://<your-account-name>.blob.core.windows.net"
+   dotenv set AZURE_STORAGE_CONTAINER_NAME "your-container-name"
+   dotenv set AZURE_STORAGE_BLOB_PREFIX "your-blob-prefix"  # Defaults to `None`
+   dotenv set AZURE_EMBEDDING_ENDPOINT "your-azure-embedding-endpoint"
+   dotenv set AZURE_CHAT_ENDPOINT "your-azure-chat-endpoint"
+   dotenv set AZURE_AI_SEARCH_ENDPOINT "your-azure-search-service-endpoint"
+   ```
+
+3. **Create vector store** (first time only):
+   This step will load documents from Azure Blob Storage and save it to the Azure AI Search vector store.
```

Comment:
It would be nice to explain that this will list blobs as documents from the container and you can control blobs enumerated over by setting the blob prefix env var.